apptail/instance.go

```diff
@@ -97,6 +119,12 @@ FORLOOP:
 				err = t.Wait()
 				break FORLOOP
 			}
+			currentOffset, err := t.Tell()
```

Comment:
> how elese can we get the current offset?

Err, yes, `t.Tell()` makes total sense here, I have no idea what I was smoking.

> Also, does t.Tell() even return errors correctly?

Ah, reading [`Tail.Tell()`](https://github.com/ActiveState/tail/blob/068b72961a6bc5b4a82cf4fc14ccc724c0cfa73a/tail.go#L122) again, it works by reusing `err`, so it does work fine.  I might be out of it when I made this comment :p